content/events/2022/12/2022-12-01-uswds-monthly-call-december-2022.md

```diff
@@ -23,6 +23,9 @@ slug: uswds-monthly-call-december-2022
 event_platform: zoom
 primary_image: 2022-uswds-monthly-call-dec-title-card
 ---
+
+{{< asset-static file="uswds-monthly-call-december-2022.pptx" label="View the slides (PowerPoint, 9.7 MB, 75 pages)" >}}
```

Comment:
@jdotyoon Can you link via the old method for now? This hasn't been uploaded to S3 bucket yet.